local_settings.sh.in

```diff
@@ -303,8 +303,36 @@ uninstall_policies() {
 	_restore_file_modes
 }
 
+verify_policies() {
+	if [[ ! selinuxenabled ]]; then
```

Comment:
I missed this the last time - you probably need to drop the brackets here:

`if ! selinuxenabled; then`

Bash is fiddly and does weird things:
```

[lhh@localhost ~]$ true
[lhh@localhost ~]$ echo $?
0
[lhh@localhost ~]$ false
[lhh@localhost ~]$ echo $?
1
[lhh@localhost ~]$ ! true
[lhh@localhost ~]$ echo $?
1
[lhh@localhost ~]$ ! false
[lhh@localhost ~]$ echo $?
0
[lhh@localhost ~]$ [[ true ]]
[lhh@localhost ~]$ echo $?
0
[lhh@localhost ~]$ [[ ! true ]]
[lhh@localhost ~]$ echo $?
1
# So far, so good...
[lhh@localhost ~]$ [[ false ]]
[lhh@localhost ~]$ echo $?
0
[lhh@localhost ~]$ [[ ! false ]]
[lhh@localhost ~]$ echo $?
1
# Uh oh!!
```

Reply:
I tested that condition here, and since we're facing only "not true", it SHOULD be fine - but I've pushed an update, just to be on the safe side.